testing/kuttl/e2e/exporter-replica/00-assert.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+# First, check that all containers in the instance(s) pod are ready
+# Then, grab the exporter metrics output and check that there were no scrape errors
```

Comment:
Are we doing this scrape errors check?

Reply:
(Also re that scrape error check: if we have the wrong password, does that metric go up?)

Reply:
nope, we should be :+1: good catch

yes, the metric will be `1` if there is an issue with the scrape, this includes incorrect password